src/frontend/src/pages/GanttPage/GanttChartComponents/GanttRequestChangeModal.tsx

```diff
@@ -86,40 +99,57 @@ export const GanttRequestChangeModal = ({ change, handleClose, open }: GanttRequ
   };
 
   return (
-    <NERModal
-      open={open}
-      onHide={handleClose}
-      title="Work Package Timeline Change Request"
-      onSubmit={handleSubmit}
-      disabled={!reasonForChange || !explanationForChange}
-      hideBackDrop
-      paperProps={{ position: 'absolute', left: -30, top: -30 }}
-    >
-      <Box sx={{ width: '250px' }}>
-        <Typography sx={{ mb: 0.5, fontSize: '1.2em' }}>{change.name}</Typography>
-        <Typography sx={{ fontSize: '1em', mb: 0.5 }}>
-          {`Old: ${changeInTimeline(change.prevStart, change.prevEnd)}`}
-        </Typography>
-        <Typography sx={{ fontSize: '1em' }}>{`New: ${changeInTimeline(change.newStart, change.newEnd)}`}</Typography>
-      </Box>
-      <Box sx={{ padding: '0 15px 0 15px', mt: 2 }}>
-        <FormControl fullWidth>
-          <InputLabel>Reason for Change</InputLabel>
-          <Select value={reasonForChange} label="Reason for Change" onChange={handleReasonChange}>
-            {Object.entries(ChangeRequestReason).map(([key, value]) => (
-              <MenuItem value={value}>{key}</MenuItem>
-            ))}
-          </Select>
-        </FormControl>
-        <TextField
-          fullWidth
-          label="Explanation for Change"
-          sx={{ mt: 2 }}
-          value={explanationForChange}
-          onChange={handleExplanationChange}
-          multiline
-        />
-      </Box>
-    </NERModal>
+    <>
+      {open && (
+        <Draggable handle=".draggable-handle">
```

Comment:
make this a new component called draggable form modal that is kind of like the ner form modal.